src/qt/bitcoingui.cpp

```diff
@@ -748,6 +748,19 @@ void BitcoinGUI::setNetworkActive(bool networkActive)
     updateNetworkState();
 }
 
+bool BitcoinGUI::updateHeadersSyncProgressLabel()
+{
+    int64_t headersTipTime = clientModel->getHeaderTipTime();
+    int headersTipHeight = clientModel->getHeaderTipHeight();
+    int estHeadersLeft = (GetTime() - headersTipTime)/600;
```

Comment:
Please don't hardcode the interblock time 600, but use Consensus::Params::nPowTargetTimespan.